apps/typegpu-docs/src/content/docs/guides/tgpu-cli.mdx

```diff
@@ -0,0 +1,521 @@
+---
+title: TypeGPU CLI
+description: A guide on how to use the TypeGPU CLI
+---
+
+import { FileTree } from '@astrojs/starlight/components';
+import FileDiff from '../../../components/FileDiff.astro';
+
+TypeGPU CLI (tgpu-cli) is a command-line interface for the TypeGPU library.
+It allows you to generate TypeGPU definitions from WGSL shaders.
```

Comment:
I would also add something like `It provides a way for a highly automated migration process and is a great tool for existing WebGPU projects, that want to start taking advantage of type-safe TypeGPU features. `